src/benchmarks/micro/MicroBenchmarks.csproj

```diff
@@ -17,6 +17,8 @@
     <EnableXlfLocalization>false</EnableXlfLocalization>
     <!-- this repo needs to build with netcoreapp3.1, so we can't use latest C# syntax -->
     <LangVersion>8.0</LangVersion>
+    <!-- Allow building with one major version, and running using a sdk with a higher major version -->
+    <RollForward>LatestMajor</RollForward>
```

Comment:
I wonder what kind of side effects it might have for non-WASM benchmarks, where afaik we have not hit this issue yet.

Perhaps you could move this change to `MicroBenchmarks.Wasm.props`:

https://github.com/dotnet/performance/blob/10d532f792cdb147c00b97f3716c4bd842449d8f/src/benchmarks/micro/MicroBenchmarks.Wasm.props#L1-L6

Reply:
I tried this, but it doesn't work because these props, and targets are imported for the wasm build, and not the first build of `MicroBenchmarks` which is used purely for running.